src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -758,9 +758,18 @@ static int rbd_nbd(int argc, const char *argv[])
 
   argv_to_vec(argc, argv, args);
   env_to_vec(args);
-  auto cct = global_init(NULL, args, CEPH_ENTITY_TYPE_CLIENT,
-			 CODE_ENVIRONMENT_DAEMON,
-			 CINIT_FLAG_UNPRIVILEGED_DAEMON_DEFAULTS);
+
+  code_environment_t code_env = CODE_ENVIRONMENT_UTILITY;
+  int env_flag = 0;
+  for (auto arg : args) {
+    if (strcmp(arg, "map") == 0) {
```

Comment:
@trociny it is a good way to separate map and other commands to handle parameters differently. But at this moment, I prefer we leave the change and fix this bug, and modify this common  global_init later.